calico_kubernetes/calico_kubernetes.py

```diff
@@ -35,6 +38,11 @@
 KUBE_API_ROOT = os.environ.get('KUBE_API_ROOT',
                                'http://kubernetes-master:8080/api/v1/')
 
+# Flag to indicate whether or not to use Calico IPAM
+# If False, use the default docker container ip address to create container
+# If True, use libcalico's auto_assign IPAM to create container
+CALICO_IPAM = os.environ.get('CALICO_IPAM', 'default')
```

Comment:
I think this should just be a boolean value.

CALICO_IPAM="true" or CALICO_IPAM="false - it should default to "false"